.github/workflows/make.yml

```diff
@@ -0,0 +1,16 @@
+name: make
+on: [pull_request, push]
+jobs:
+  make:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: sudo apt-get -q update
+      - run: sudo apt-get -qq upgrade
+      - run: sudo apt-get -qq install gettext python3 python3-pip python3-babel python3-boto3 python3-jinja2
+                                      python3-numpy python3-pillow python3-pycurl python3-tornado
+      - run: cd motioneye && make
+      - run: pip install build && python3 -m build  # Stores .tar.gz and .whl files into ./dist
+      # - run: python3 -m pip install https://github.com/motioneye-project/motioneye/archive/dev.zip
```

Comment:
It should be definitely done on the related branch, not hardcoded the `dev` branch. Another way I'd suggest is, instead of installing the required Python modules via APT, we should do it via pip, using the [requirements.txt](https://github.com/motioneye-project/motioneye/blob/dev/requirements.txt). That way it becomes OS independent as well, and we do not need to change dependencies in two places.

Also it makes the time consuming `apt-get upgrade` obsolete, I mean not really, but if we do not explicitly need to install a new APT package, I'd skip it.

Reply:
`pip install` is now compatible with pyproject.toml + setup.cfg and no longer needs a requirements.txt.